docs/rest.md

```diff
@@ -192,6 +195,13 @@ Here is one way you might customize `RestLink`:
         bodySnippet...
       }
     },
+    bodySerializer: (body: any) => {
```

Comment:
I think this option should be `defaultSerializer` now. Should it also be receiving the `headers` as a second parameter and returning `{body, headers}`?

Reply:
Yep you are right!